Doc/library/copyreg.rst

```diff
@@ -1,5 +1,5 @@
-:mod:`copyreg` --- Register :mod:`pickle` support functions
-===========================================================
+:mod:`!copyreg` --- Register :mod:`!pickle` support functions
+=============================================================
```

Comment:
This link _is_ useful, as it takes you to the documentation for another module:

```suggestion
:mod:`!copyreg` --- Register :mod:`pickle` support functions
============================================================
```

Reply:
Yes, this was one of the edge cases.  The pickle module is linked in the first paragraph, which is more useful in any case.

Reply:
True. I'm okay with it just being linked in the first paragraph.